android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabWindowFeature.kt

```diff
@@ -83,18 +37,7 @@ class CustomTabWindowFeature(
                 .collect { state ->
                     val windowRequest = state.content.windowRequest
                     if (windowRequest?.type == WindowRequest.Type.OPEN) {
-                        val intent = configToIntent(state.config)
-                        val uri = windowRequest.url.toUri()
-                        // This could only fail if the above intent is for our application
-                        // and we are not registered to handle its schemes.
-                        // Let's log this to better asses how often this happens in real world and
-                        // if we need to add new schemes to properly support this workflow.
-                        // See Fenix #8412
-                        try {
-                            intent.launchUrl(activity, uri)
-                        } catch (e: ActivityNotFoundException) {
-                            onLaunchUrlFallback(uri)
-                        }
+                        state.engineState.engineSession?.loadUrl(windowRequest.url)
```

Comment:
This part is the main change.  The rest of the changes is just that we don't need `configToIntent` function anymore.

Reply:
@rocketsroger I've left a comment in [the original bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1832357#c15) that this change has the possibility of breaking a lot of web content use cases.

Reply:
I'll make sure I look into that so we won't break anything in custom tabs.

Reply:
In case there is some misunderstanding, I am suggesting that we are going to break **web content** with this change, not the custom tabs API, implementation, or UI.

Reply:
Yes, what I meant was that this can only break web content in the custom tabs.  Since this code only handles the new window request on custom tabs.

That said, there is no reason for me to imitate what Chrome does in custom tab to fix this issue.  I'll update this change so we only open the link in the current tab if we know that the url scheme is not supported in a new instance of a custom tab.